nuntium/popit_api_instance.py

```diff
@@ -1,9 +1,34 @@
 from popit.models import ApiInstance, Person, get_paginated_generator
 from contactos.models import Contact
 from mailit import MailChannel
+from datetime import datetime
 
 
-class PopitPerson(Person):
+class IsActiveMixin(object):
+    format = "%Y-%m-%d"
```

Comment:
This feels like a slightly dangerous attribute name to add to the mixin. You can imagine wanting to use this mixin on something else that already had a `format` attribute to use for something else.